group_vars/all/vars.sample

```diff
@@ -45,6 +45,7 @@ windows_update: true
 # Windows server timezone
 # Important that this value matches the timezone used by UCP servers for certificate validation
 windows_tz: 'Pacific Standard Time'
+windows_ntp_servers: "10.60.1.123 0.us.pool.net.org 1.us.pool.net.org"
```

Comment:
There is no reason to define a special variable for the list of ntp servers for Windows.  You should use the parameter defined in group_vars/all/vars.

Reply:
Yup, good call.  I had not played with the jinja2 scripting stuff but I eventually figured out how to convert the existing ntp_servers variable into the format Windows requires.  I've removed windows_ntp_servers from both vars and vars.sample.